Volume/Models/Publication.swift

```diff
@@ -17,8 +17,13 @@ struct Publication: Hashable, Identifiable {
     let backgroundImageUrl: URL?
     let recent: String?
     let shoutouts: Int
-    let socials: [String : String] = [:]
     let websiteUrl: URL?
+    let socials: [Social]?
```

Comment:
I would make this not optional. It seems from `init` that it will always be set to something not-nil. In the case where there are no socials, it should be an empty array.